packages/@react-spectrum/tooltip/test/TooltipTrigger.test.js

```diff
@@ -10,95 +10,671 @@
+    it('does not allow more than one tooltip open at a time', () => {
+      let helpfulText = 'Helpful information.';
+      let unHelpfulText = 'Unhelpful information.';
+      let {getByLabelText, getByText} = render(
+        <Provider theme={theme}>
+          <TooltipTrigger>
+            <ActionButton aria-label="good-trigger" />
+            <Tooltip>{helpfulText}</Tooltip>
+          </TooltipTrigger>
+          <TooltipTrigger>
+            <ActionButton aria-label="bad-trigger" />
+            <Tooltip>{unHelpfulText}</Tooltip>
+          </TooltipTrigger>
+        </Provider>
+      );
 
-      await waitFor(() => {
-        expect(tooltip).not.toBeInTheDocument();
+      let goodButton = getByLabelText('good-trigger');
+      let badButton = getByLabelText('bad-trigger');
+      act(() => {
+        goodButton.focus();
       });
+      expect(getByText(helpfulText)).toBeVisible();
+
+      // we've used focus to open one, now we can use hover to try and open a second one
+      fireEvent.mouseEnter(badButton);
+      fireEvent.mouseMove(badButton);
+
+      expect(() => getByText(helpfulText)).toThrow();
+      expect(getByText(unHelpfulText)).toBeVisible();
+    });
+
+    it('two can be open at once if in controlled open state', () => {
+      let helpfulText = 'Helpful information.';
+      let unHelpfulText = 'Unhelpful information.';
+      let {getByLabelText, getByText} = render(
+        <Provider theme={theme}>
+          <TooltipTrigger isOpen>
+            <ActionButton aria-label="good-trigger" />
+            <Tooltip>{helpfulText}</Tooltip>
+          </TooltipTrigger>
+          <TooltipTrigger>
+            <ActionButton aria-label="bad-trigger" />
+            <Tooltip>{unHelpfulText}</Tooltip>
+          </TooltipTrigger>
+        </Provider>
+      );
+
+      let badButton = getByLabelText('bad-trigger');
+      expect(getByText(helpfulText)).toBeVisible();
+
+      // open a second one through interaction
+      fireEvent.mouseEnter(badButton);
+      fireEvent.mouseMove(badButton);
+
+      expect(getByText(helpfulText)).toBeVisible();
+      expect(getByText(unHelpfulText)).toBeVisible();
+    });
+  });
+  describe('disabled', () => {
+    it('can be disabled from the TooltipTrigger', () => {
+      let {getByRole, getByLabelText} = render(
+        <Provider theme={theme}>
+          <TooltipTrigger isDisabled>
+            <ActionButton aria-label="trigger" />
+            <Tooltip>Helpful information.</Tooltip>
+          </TooltipTrigger>
+        </Provider>
+      );
+      let button = getByLabelText('trigger');
+      fireEvent.mouseEnter(button);
+      fireEvent.mouseMove(button);
+
+      expect(() => getByRole('tooltip')).toThrow();
+    });
+
+    // Given that a disabled button cannot be focused, i assume we want to stop this.
+    // isDisabled on TooltipTrigger disables the tooltip, not the Button, this one makes some sense, though maybe a bad prop name?
```

Comment:
o yeah, here's the comment i was looking for in 
>I really thought i'd written a comment about this somewhere. Here's the dilemma
If I disable the button, which is totally valid, I don't know about it in the trigger.
Should we disable all mouse events on disabled buttons/inputs?
pointer-events: none;

>disabled makes it so that we can't focus it, so I think it makes sense that you shouldn't be able to hover it either?

>It makes sense for a tooltiptrigger to be disabled but have the button still be interact-able, maybe the user has some setting to turn off tooltips so that they don't appear. At least, I think this makes sense.

Reply:
cool I think what you have right now is good, might be worth adding a story for having `isDisabled` on the TooltipTrigger to show that the button is still interact-able and that the tooltip doesn't show up.